builders/get-started/networks/moonbeam-dev.md

```diff
@@ -202,53 +201,117 @@ Flags do not take an argument. To use a flag, add it to the end of a command. Fo
+- **`--ws-max-connections <connections>`** - specifies the maximum number of WS RPC server connections. As of [client v0.30.0](https://github.com/PureStake/moonbeam/releases/tag/v0.30.0){target=_blank}, this flag adjusts the combined HTTP and WS connection limit. Default is 100 connections
 
 For a complete list of flags and options, spin up your Moonbeam development node with `--help` added to the end of the command.
 
-## Debug, Trace and TxPool APIs {: #debug-trace-txpool-apis } 
-
-You can also gain access to some non-standard RPC methods by running a tracing node, which allow developers to inspect and debug transactions during runtime. Tracing nodes use a different Docker image than a standard Moonbeam development node. 
+## Prefunded Development Accounts {: #pre-funded-development-accounts } 
 
-To learn how to run a Moonbeam development tracing node, check out the [Run a Tracing Node](/node-operators/networks/tracing-node){target=_blank} guide and be sure to switch to the **Moonbeam Development Node** tab throughout the instructions. Then to access the non-standard RPC methods with your tracing node, check out the [Debug & Trace](/builders/build/eth-api/debug-trace){target=_blank} guide.
+Moonbeam has a [unified accounts](/learn/features/unified-accounts){target=_blank} system, which enables users to have an Ethereum-styled H160 account that can interact with the Substrate API and the Ethereum API. As a result, you can interact with your account through [Polkadot.js Apps](#connecting-polkadot-js-apps-to-a-local-moonbeam-node) or [MetaMask](/tokens/connect/metamask){target=_blank} (or any other [EVM wallet](/tokens/connect/){target=_blank}). In addition, you can also use other [development tools](/builders/build/eth-api/dev-env/){target=_blank}, such as [Remix](/builders/build/eth-api/dev-env/remix/){target=_blank} and [Hardhat](/builders/build/eth-api/dev-env/hardhat/){target=_blank}.
 
-## Pre-funded Development Accounts {: #pre-funded-development-accounts } 
-
-Your Moonbeam development node comes with ten pre-funded accounts for development. The addresses are derived from Substrate's canonical development mnemonic: 
+Your Moonbeam development node comes with ten prefunded Ethereum-styled accounts for development. The addresses are derived from Substrate's canonical development mnemonic: 
 
 ```
 bottom drive obey lake curtain smoke basket hold race lonely fit walk
 ```
 
---8<-- 'code/setting-up-node/dev-accounts.md'
+??? note "Development account addresses and private keys"
+    --8<-- 'code/setting-up-node/dev-accounts.md'
+
+    Also included with the development node is an additional prefunded account used for testing purposes:
```

Comment:
```suggestion
Also included with the development node is an additional prefunded account used for testing purposes:
```

Reply:
If you leave the space this is how it is rendered which is weird:

![image](https://user-images.githubusercontent.com/64150856/232565882-f8354941-9b0a-4a74-be8b-a1740982d5a2.png)

Reply:
Ahh and please also remove the spaces in the code snippet. So it should be:

```
Also included with the development node is an additional prefunded account used for testing purposes:

--8<-- 'code/setting-up-node/dev-testing-account.md'
```